src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -470,6 +470,7 @@ void GcsPlacementGroupManager::LoadInitialData(const EmptyCallback &done) {
   auto callback = [this,
                    done](const std::unordered_map<PlacementGroupID,
                                                   rpc::PlacementGroupTableData> &result) {
+    std::unordered_map<NodeID, std::vector<rpc::Bundle>> node_to_bundles;
     for (auto &item : result) {
       auto placement_group = std::make_shared<GcsPlacementGroup>(item.second);
       if (item.second.state() != rpc::PlacementGroupTableData::REMOVED) {
```

Comment:
Btw, why do we add removed placement groups to the entry? I think we currently delete placement groups when they are deleted.

Reply:
We filtered out the removed placement groups.

Reply:
_Oops..! lol. Sorry I was confused...